package.json

```diff
@@ -164,11 +164,18 @@
       ],
       "order": 8
     },
+    "cleanAfterBuild": {
+      "title": "Clean after Build",
+      "description": "Run clean command after every successful build.",
```

Comment:
The default settings for `cleanPatterns` perform a deep clean, i.e. delete all DVI, PS, PDF and synctex.gz files. If the user enables `cleanAfterBuild` without changing the default patterns then the build will execute and everything will be deleted.

At a minimum we need some language here explaining that those extensions should probably be removed from `cleanPatterns`. At some point we might consider checking the pattern list explicitly, but it seems to me we would need the explanation here in any case.